rackcity/views/asset_views.py

```diff
@@ -420,6 +506,7 @@ def asset_bulk_upload(request):  # need to make network and power connections he
     for asset_to_add in assets_to_add:
         records_added += 1
         asset_to_add.save()
+        # will need to save mac addresses and connections here
```

Comment:
Didn't address bulk stuff in this PR because we'll get mac addresses in a very different format from bulk

Reply:
💪